src/transformers/models/wav2vec2_conformer/modeling_wav2vec2_conformer.py

```diff
@@ -856,8 +923,12 @@ def __init__(self, config):
         else:
```

Comment:
Well more precisely, there is one position embedding layer per encoder layer (identified by `distance_embedding`). Because of the small context size (thanks to relative position), it doesn't take that much space